packages/v4/patternfly-docs/content/design-guidelines/components/text-input-group/text-input-group.md

```diff
@@ -0,0 +1,38 @@
+---
+id: Text input group
+section: components
+---
+
+A **text input group** is a more flexible composable version of a [text input](/components/text-input). It enables consumers of PatternFly to build custom inputs for filtering and similar use cases by placing elements like icons, [chips groups](components/chip-group) and [buttons](components/button) within a text input.
+
+## Elements
+
+<img src="./img/text-input-elements.png" alt="text input elements" width="509"/>
+
+1. **Icon (optional):** An icon can be used in front of the input field to identify the purpose of the text input.
+
+2. **Input field:** The input field itself can be customized to use multiple text styles as shown here. This styling is not possible using a standard text input.
+
+3. **Clear button (optional):** A clear button can be added to clear the contents of the field.
+
+4. **Chip group (optional):** A chip group can be nested inside of the text input as shown here.
+
+## Usage
+
+The text input group component is useful for building more complex custom inputs. It's usage includes, but is not limited to, the following use cases. Each of these can be found in a [React demo here](/components/text-input-group/react-demos).
```

Comment:
Typo - should be "its" usage, not "it's" usage

Reply:
Would also delete the "here" at the end of the sentence, and just link to the "React demo" or say "can be found in this React demo" ..